### ISSUE TITLE
Plugin updater should delete old .pdb

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#631]
I'm found StdChat.pdb  in Miranda directory yesterday. But StdChat.dll was deleted much earlier.
I think, plugin updater at every update should delete all .pdb who do not have the corresponding .dll and .exe (in installation directory, /Plugins and /Core),

**Comment by Robyer** [comment]
Also it should delete all .pdb files of updated plugins.

So, any file that PU updates or deletes, it should check if there exists corresponding .pdb file and delete it as well.

**Comment by dartraiden** [comment]
And maybe better move all .pdb into separate folder? [User ask for this](http://forum.miranda-ng.org/index.php?topic=5239.msg19981).

**Comment by Robyer** [comment]
@dartraiden I'd want that too. It's so painful to have them in main directory with *.exe and other files.

**Comment by LittleVulpix** [comment]
Yes, I would also like PDBs to be in their own directory. Maybe something like Miranda NG\Plugins\Symbols
